src/api/v1/event/index.js

```diff
@@ -130,4 +132,25 @@ router.route('/')
     }).catch(next);
   });
 
+/**
+ * Uploads a profile picture for a given event. Requires admin access
+ */
+router.post('/picture', Storage.bufferImageBlob(Media.mediaTypes.EVENT_COVER), async (req, res, next) => {
+  if (!req.user.isAdmin()) return next(new error.Forbidden());
+  if (!req.body.uuid) return next(new error.BadRequest('UUID must be provided in body'));
+  try {
+    const cover = await Storage
+      .uploadToS3(Media.mediaTypes.EVENT_COVER, req.file, req.body.uuid)
+      .then((data) => data.Location)
```

Comment:
Move this to inside the `upload` function since the controller expects the function to upload the image and return the location, not some random object from the AWS SDK that has the location as a property. The storage service should handle anything related to S3, including pulling the location from the AWS SDK object.